frontend/packages/dev-console/integration-tests/features/topology/topology_pipeline_feature-ODC-3925

```diff
@@ -0,0 +1,31 @@
+Feature: Pipeline features in topology
+	As a user, I want to check pipeline features in topology
+
+Background:
+    Given user is at developer perspective
+    And user has installed Openshift Pipelines Operator
+    And user has selected namespace "aut-topology-pipeline"
+
+Examples:
+    | resource_type     | workload_name   |
+    | deployment        | nodejs-ex-git-1 |
+    | deployment config | dancer-ex-git-1 |
+
+
+@regression
+Scenario Outline: Sidebar for workload
+   Given user created workload "<workload_name>" with resource type "<resource_type>" with pipeline 
+   And pipeline has not autostarted
+   When user clicks on workload "<workload_name>"
+   Then user can see sidebar opens with Resources tab selected by default
+   And user can see Pipeline Runs section
+   And user can not see Buildconfig Runs section   
```

Comment:
```suggestion
    And user can see an pipeline not started alert 
    And user can not see Buildconfig Runs section   
```

Reply:
Refer https://github.com/openshift/console/pull/7241 for more details
![](https://user-images.githubusercontent.com/9964343/99662727-a612c200-2a8b-11eb-9986-17a211bdc6ff.png)

Reply:
Updated